src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -227,12 +227,17 @@ const generateTaskRole = (taskRole, config) => {
   return frameworkTaskRole;
 };
 
-const generateFrameworkDescription = (frameworkName, config) => {
+const generateFrameworkDescription = (frameworkName, userName, config, rawConfig) => {
   const frameworkDescription = {
     apiVersion: launcherConfig.apiVersion,
     kind: 'Framework',
     metadata: {
       name: frameworkName,
+      annotations: {
```

Comment:
Make the userName, virtualCluster also be the label for framework K8s object ?

Reply:
added